lib/matplotlib/widgets.py

```diff
@@ -1829,11 +1831,11 @@ def __init__(self, ax, onselect, useblit=False, button=None,
         self._eventrelease = None
         self._prev_event = None
         self._state = set()
-        self._default_state = set()
 
     eventpress = _api.deprecate_privatize_attribute("3.5")
     eventrelease = _api.deprecate_privatize_attribute("3.5")
     state = _api.deprecate_privatize_attribute("3.5")
+    state_modifier_keys = _api.deprecate_privatize_attribute("3.5")
```

Comment:
```suggestion
    state_modifier_keys = _api.deprecate_privatize_attribute("3.6")
```
AFAICS this is new. So we must target the next minor release. This also needs a deprecation note in `doc/api/next_api_changes/deprecations`